compass/ocean/mesh/remap_topography.py

```diff
@@ -94,64 +108,203 @@ def run(self):
-                                           'renorm_threshold')
-        ice_density = config.getfloat('remap_topography', 'ice_density')
-        ocean_density = constants['SHR_CONST_RHOSW']
-        g = constants['SHR_CONST_G']
+        if method != 'conserve':
+            raise ValueError(f'Unsupported method {method} for TempestRemap')
+
+        args = [
+            'mbtempest', '--type', '5',
+            '--load', f'source.scrip.64bit.p{self.ntasks}.h5m',
+            '--load', f'target.scrip.64bit.p{self.ntasks}.h5m',
+            '--file', f'map_source_to_target_{method}.nc',
+            '--weights', '--gnomonic',
+            '--boxeps', '1e-9',
+        ]
+
+        run_command(
+            args, self.cpus_per_task, self.ntasks,
+            self.openmp_threads, self.config, self.logger,
+        )
+
+        logger.info('  Done.')
+
+    def _create_weights_esmf(self):
+        """
+        Create mapping weights file using ESMF_RegridWeightGen
+        """
+        logger = self.logger
+        logger.info('Create weights file')
 
-        in_descriptor = LatLonGridDescriptor.read(fileName='topography.nc',
-                                                  lonVarName=lon_var,
-                                                  latVarName=lat_var)
+        config = self.config
+        method = config.get('remap_topography', 'method')
+
+        args = [
+            'ESMF_RegridWeightGen',
+            '--source', 'source.scrip.nc',
+            '--destination', 'target.scrip.nc',
+            '--weight', f'map_source_to_target_{method}.nc',
+            '--method', method,
+            '--netcdf4',
+            '--ignore_unmapped',
+        ]
 
-        in_mesh_name = in_descriptor.meshName
+        run_command(
+            args, self.cpus_per_task, self.ntasks,
+            self.openmp_threads, self.config, self.logger,
+        )
 
-        out_mesh_name = self.mesh_name
-        out_descriptor = MpasCellMeshDescriptor(fileName='base_mesh.nc',
-                                                meshName=self.mesh_name)
+        logger.info('  Done.')
+
+    def _remap_to_target(self):
+        """
+        Remap combined bathymetry onto MPAS target mesh
+        """
+        logger = self.logger
+        logger.info('Remap to target')
 
-        mapping_file_name = \
-            f'map_{in_mesh_name}_to_{out_mesh_name}_{method}.nc'
-        remapper = Remapper(in_descriptor, out_descriptor, mapping_file_name)
+        config = self.config
+        method = config.get('remap_topography', 'method')
 
-        remapper.build_mapping_file(method=method, mpiTasks=self.ntasks,
-                                    tempdir='.', logger=logger,
-                                    esmf_parallel_exec=parallel_executable)
+        # Build command args
+        # Unused options:
+        # -P mpas, handles some MPAS-specific index ordering, CF, etc...
+        # -C climatology, basically bypasses fill values
```

Comment:
Let's remove these comments.  They were probably helpful for your learning process but aren't going to be useful to the average developer.